src/VisualStudio/CSharp/Test/ProjectSystemShim/LegacyProject/AnalyzersTests.cs

```diff
@@ -244,7 +244,7 @@ public async Task RuleSet_FileChangingOnDiskRefreshes(bool useCpsProject)
             using var environment = new TestEnvironment();
             if (useCpsProject)
             {
-                await CSharpHelpers.CreateCSharpCPSProjectAsync(environment, "Test", binOutputPath: null, $"/ruleset:\"{ruleSetFile.Path}\"");
+                await CSharpHelpers.CreateCSharpCPSProjectAsync(environment, "Test", binOutputPath: @"C:\test.dll", $"/ruleset:\"{ruleSetFile.Path}\"");
```

Comment:
Did this break if we had a null path? We absolutely have legacy projects that get created for intellisense projects in certain cases for legacy ASP.NET; I don't think that'd go through the CPS case but that is worrisome to me. If we're making stronger assertions definitely go and do a DDRIT run first.

Reply:
Will test.